### PR TITLE
Updating zkSync Era information

### PR DESCRIPTION
**Pull request by githubdoramon**
This PR will be valid on March 24th. I'll change its state then

### CODE REVIEW

File: packages/config/src/layer2s/zksyncera.ts [hunk 1]
```diff
@@ -21,15 +21,14 @@ export const zksyncera: Layer2 = {
   display: {
     name: 'zkSync Era',
     slug: 'zksync-era',
-    warning: 'Currently only registered addresses can bridge to zkSync Era.',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/zksyncera.ts:24]: Is this true? I'm trying to bridge to zksync era using https://bridge.zksync.io/ and I keep getting this prompt 
Screenshot 2023-03-23 at 11 59 26

Comment (githubdoramon) [reply packages/config/src/layer2s/zksyncera.ts:24]: @michalsidzej this PR is supposed to go live tomorrow :) - Then, it will be true

Comment (michalsidzej) [reply packages/config/src/layer2s/zksyncera.ts:24]: Cool, make sure you include it into description or mark the PR as draft :) Thanks for contributing tho 👍

Comment (githubdoramon) [reply packages/config/src/layer2s/zksyncera.ts:24]: good point... added now, and will change tomorrow

**Comment by githubdoramon** [comment]
@michalsidzej PR is now valid and ready for review :)

**Pull request merged**